### PR TITLE
Added new command line argument to run queries from file

### PR DESCRIPTION
**Pull request by nbhavana**

### CODE REVIEW

File: pkg/query/query.go [hunk 1]
```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
 		_ = sendSingleRequest(freeText, client, fQuery, requestStr, true)
 	}
 }
+
+// Run queries from a csv file
+
+func RunQueryFromFile(dest string, numIterations int, prefix string, continuous, verbose bool, filepath string) {
+	// open file
+	f, err := os.Open("queries.csv")
```
Comment (srisub98) [inline pkg/query/query.go:415]: is this supposed to be `filepath` instead of hard coded string?

File: pkg/query/query.go [hunk 2]
```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
...
+	if err != nil {
+		log.Infof("RunQueryFromFile: Error in opening file: %v, err: %v", filepath, err)
+	}
```
Comment (srisub98) [inline pkg/query/query.go:418]: return `if err != nil`

File: pkg/query/query.go [hunk 3]
```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
...
+	// read csv values using csv.Reader
+	csvReader := csv.NewReader(f)
+	for {
+		rec, err := csvReader.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			log.Infof("RunQueryFromFile: Error in reading file: %v, err: %v", filepath, err)
+		}
```
Comment (srisub98) [inline pkg/query/query.go:431]: nit: same here, on errors we should return

File: pkg/query/query.go [hunk 4]
```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
...
+		// do something with read line
+		fmt.Printf("%+v\n", rec)
```
Comment (srisub98) [inline pkg/query/query.go:433]: use the already imported `log` module & maybe we can put such statements behind the `verbose` flag

File: pkg/query/query.go [hunk 5]
```diff
@@ -401,3 +407,87 @@ func runContinuousQueries(client *http.Client, requestStr string) {
...
+								if actualHits != rec[1] {
+									log.Fatalf("Actual Hits: %v does not match Expected hits: %v for query:%v", actualHits, rec[1], rec[0])
+								} else {
+									log.Infof("Query %v was succesful", rec[0])
+								}
+
+							}
+						}
+					}
+				}
```
Comment (srisub98) [inline pkg/query/query.go:489]: nit: if we don't see `k == totalMatched`, we would not error out although the returned response is not accurate 

also, we should keep separate variable naming for inner & outer loops

Comment (nbhavana) [reply pkg/query/query.go]: what do we want to do if if we don't see k == totalMatched?  don't we always return totalMatched in Complete state?

**Review by srisub98 — APPROVED** [review]

**Pull request merged**